### DISCUSSION TITLE
Assumed Bitbucket-Server Repository URL may be wrong (context path)

### DISCUSSION THREAD

**Discussion by thriqon** [renovatebot/renovate#33012] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

v38.142.7 / Bitbucket Server

### Please tell us more about your question or problem

When using bitbucket-server, renovate and git-url-parse assume that `https://user:password@server.local/scm/` is a valid replacement via `
url.<base>.insteadOf ` to access `ssh://server.local:7999/`

Problem is: bitbucket-server may be operated with a context path, e.g. `https://server.local/bitbucket/scm`.

I found no way to configure this prefix, only code: #29527 and https://github.com/renovatebot/renovate/blob/c9521dbc4400331b9aa06d281ddc55aa1b90599d/lib/util/git/url.ts#L42

Do you have any idea on how to proceed here?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment]
not supported scenario
